proto/google/api/annotations.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015, Google Inc.
```

Comment:
Git submodules + npm is not a great story. It would be great if the proto files could be published to npm, and we could treat them as a normal dependency. Do you think there's any chance for that?